android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java

```diff
@@ -212,9 +212,12 @@ public void dismiss() {
      * @param tags
      */
     @ReactMethod
-    public void appendTags(String tags) {
+    public void appendTags(ReadableArray tags) {
         try {
-            String[] result = tags.split(",");
+            String[] result = new String[tags.size()];
+            for(int i = 0; i < tags.size(); i++) {
```

Comment:
@salmatarzi Instead of converting this yourself you can use `ArrayUtil.toArray(ReadableArray readableArray);`